.gitlab/internal_kubernetes_deploy.yml

```diff
@@ -58,5 +58,7 @@ notify-slack:
   tags: ["arch:amd64"]
   needs: ["internal_kubernetes_deploy_experimental"]
   script:
+    - SDM_JWT=$(vault read -field=token identity/oidc/token/sdm)
+    - export SDM_JWT
```

Comment:
```suggestion
    - export SDM_JWT=$(vault read -field=token identity/oidc/token/sdm)
```

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe a bit shorter, WDYT ?

Reply:
sure, thanks!